File: basics/integer-comparison/python/integer-comparison.py
import sys

def main():
    i1 = sys.argv[1]
    i2 = sys.argv[2]
    if i1 < i2: print('%s is less than %s.' % (i1, i2))
    if i1 == i2: print('%s is equal to %s.' % (i1, i2))
    if i1 > i2: print('%s is greather than %s.' % (i1, i2))

if __name__ == "__main__":
    main()

File: basics/integer-comparison/ruby/IntegerComparison.rb
i1 = ARGV[0].to_i
i2 = ARGV[1].to_i

if (i1 < i2) then
  puts "#{i1} is less than #{i2}."
elsif (i1 == i2)
  puts "#{i1} is equal to #{i2}."
else
  puts "#{i1} is greater than #{i2}."
end
